supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -11,11 +11,11 @@ For documentation questions, contact rhos-docs@redhat.com.
 [discrete]
 [[director]]
 ==== image:images/yes.png[yes] director (noun)
-*Description*: The Red Hat OpenStack Platform director is a toolset for installing and managing a complete OpenStack environment. Write in lower case. Use with the article 'the' where appropriate. For example: _"Create an OpenStack environment with the director."_
+*Description*: The Red Hat OpenStack Platform (RHOSP) director is a toolset for installing and managing a complete OpenStack environment. Write in lowercase. For example: _Use director to create a RHOSP environment._
```

Comment:
Does one pronounce RHOSP as one word or as a series of letters? I would pronounce the word in a series of letters, but this could just be me :) . 

The context of my query is around the indefinite article choice. I see an answer further down, so I'll take it RHSOP Is pronounced as one word?